src/lib/utils/index.ts

```
@@ -22,13 +22,40 @@ export const omit = (target: Record<string, unknown>, path: string[]) => {
+
+  paramsObject = pick(paramsObject, rootParams) as Record<
+    string,
+    string | number
+  >;
+  paramsObject.authUrlParams = omit(
+    paramsObject.authUrlParams as Record<string, unknown>,
+    rootParams,
+  );
+
+  if (Object.keys(paramsObject.authUrlParams).length === 0) {
+    delete paramsObject.authUrlParams;
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)